### ISSUE TITLE
If hooks config is changed with vim, -hotreload loses track of the file

### ISSUE THREAD

**Issue by ivanpesin** [adnanh/webhook#171]
Steps to reproduce:

- start webhook with hotreload
- change config with `vim` 
- webhook says config file was removed, not updated

```
 $ ./webhook -verbose -hotreload
[webhook] 2017/09/10 23:16:13 version 2.6.5 starting
[webhook] 2017/09/10 23:16:13 setting up os signal watcher
[webhook] 2017/09/10 23:16:13 os signal watcher ready
[webhook] 2017/09/10 23:16:13 attempting to load hooks from hooks.json
[webhook] 2017/09/10 23:16:13 found 2 hook(s) in file
[webhook] 2017/09/10 23:16:13   loaded: webhook
[webhook] 2017/09/10 23:16:13   loaded: sleep
[webhook] 2017/09/10 23:16:13 setting up file watcher for hooks.json
[webhook] 2017/09/10 23:16:13 serving hooks on http://0.0.0.0:9000/hooks/{id}
[webhook] 2017/09/10 23:16:22 hooks file hooks.json removed, no longer watching this file for changes, and removing hooks that were loaded fromit
[webhook] 2017/09/10 23:16:22   removing: webhook
[webhook] 2017/09/10 23:16:22   removing: sleep
[webhook] 2017/09/10 23:16:22 removed 2 hook(s) that were loaded from file hooks.json
```

I'm going to look if I can fix it, but I wanted to check why webhook stops watching for a file once it is removed? Wouldn't it make sense to continue watching and load configuration if the file is created again? That would've fixed the `vim` issue.

**Comment by gdubicki** [comment]
Isn't this the same as #150 ?

**Comment by adnanh** [comment]
Yes, it's a duplicate of #150

**Comment by adnanh** [comment]
Fixed in v2.6.6 by #172
